kubernetes/freeradius/templates/PersistentVolume.yaml

```
@@ -0,0 +1,19 @@
+{{- if and .Values.persistence.enabled .Values.persistence.existingClaim }}
+{{- $namespace := .Release.Namespace }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)